sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.institutions.controller.saveScholasticStanding.e2e-spec.ts

```diff
@@ -875,6 +907,339 @@ describe("StudentScholasticStandingsInstitutionsController(e2e)-saveScholasticSt
       });
   });
 
+  // Full-time related student restrictions.
+  describe(`Full-time restrictions for '${StudentScholasticStandingChangeType.StudentDidNotCompleteProgram}'.`, () => {
+    it(`Should create an ${RestrictionCode.SSR} restriction when the student exceeds the maximum number of unsuccessful weeks application.`, async () => {
+      // Arrange
+      const unsuccessfulWeeks =
+        SCHOLASTIC_STANDING_MINIMUM_UNSUCCESSFUL_WEEKS + 1;
+      mockFormioDryRun({
+        validDryRun: true,
+        studentScholasticStandingChangeType:
+          StudentScholasticStandingChangeType.StudentDidNotCompleteProgram,
+        numberOfUnsuccessfulWeeks: unsuccessfulWeeks,
+      });
+      const application = await saveFakeApplication(
+        db.dataSource,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          offeringIntensity: OfferingIntensity.fullTime,
+          applicationStatus: ApplicationStatus.Completed,
+          offeringInitialValues: {
+            studyBreaks: {
+              totalDays: unsuccessfulWeeks * 7,
+            } as StudyBreaksAndWeeks,
+          },
+        },
+      );
+      // Institution token.
+      const institutionUserToken = await getInstitutionToken(
+        InstitutionTokenTypes.CollegeFUser,
+      );
+      const endpoint = `/institutions/scholastic-standing/location/${collegeFLocation.id}/application/${application.id}`;
+
+      // Act/Assert
+      await request(app.getHttpServer())
+        .post(endpoint)
+        .send(payload)
+        .auth(institutionUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.CREATED);
+      const studentRestrictions = await getActiveStudentRestrictions(
+        application.student.id,
+      );
+      expect(studentRestrictions).toEqual([
+        {
+          id: expect.any(Number),
+          restriction: {
+            id: ssrRestriction.id,
+            restrictionCode: RestrictionCode.SSR,
+          },
+        },
+      ]);
+    });
+
+    it(`Should create an ${RestrictionCode.SSRN} restriction when the student already has an ${RestrictionCode.SSR}.`, async () => {
+      // Arrange
+      const unsuccessfulWeeks =
+        SCHOLASTIC_STANDING_MINIMUM_UNSUCCESSFUL_WEEKS + 1;
```

Comment:
The unsuccessful weeks can be just 1 to validate that it doesn't wait for 68 weeks of UC to add SSRN.

Reply:
Removed the `SCHOLASTIC_STANDING_MINIMUM_UNSUCCESSFUL_WEEKS` since its default value will be 1.